core/ui/src/main/kotlin/com/nexters/bandalart/android/core/ui/component/BottomSheetButton.kt

```diff
@@ -0,0 +1,44 @@
+package com.nexters.bandalart.android.core.ui.component
+
+import android.widget.Toast
+import androidx.compose.foundation.layout.height
+import androidx.compose.material3.FilledIconButton
+import androidx.compose.material3.IconButtonColors
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.unit.dp
+import com.nexters.bandalart.android.core.ui.theme.Gray200
+import com.nexters.bandalart.android.core.ui.theme.Gray400
+import com.nexters.bandalart.android.core.ui.theme.Gray900
+import com.nexters.bandalart.android.core.ui.theme.White
+
+@Composable
+fun BottomSheetDeleteButton(
+  modifier: Modifier = Modifier,
+) {
+  val contextForToast = LocalContext.current.applicationContext
```

Comment:
해당 함수내에서 다른 context (ActivityContext) 를 쓰지 않으니까 굳이 context 를 변수명으로 구분할 필요는 없을 것 같아 
```suggestion
val context = LocalContext.current.applicationContext
```